src/lib/domains/agentic/services/chat-strategies/relevant.strategy.ts

```
@@ -0,0 +1,76 @@
+      },
+      serialize: (format) => this.serialize(contextMessages, format)
+    }
+  }
+  
+  private extractTextContent(content: string | ChatWidget): string {
+    if (typeof content === 'string') return content
+    
+    // Handle widget content extraction
+    switch (content.type) {
+      case 'preview':
+        const previewData = content.data as { title?: string; content?: string }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)